src/parametric.cpp

```diff
@@ -984,7 +1230,7 @@ void ParametricViewer::SaveToCSV()
 
 }
 
-void ParametricViewer::SendToExcel()
+void ParametricViewer::SendToExcelOld()
```

Comment:
Same note as SaveToCSVOld

Reply:
All old functions removed in the last commit. Thanks for the detailed review